packages/backend/src/open_payments/payment/incoming/model.ts

```diff
@@ -68,11 +69,20 @@ export class IncomingPayment
         from: 'incomingPayments.assetId',
         to: 'assets.id'
       }
+    },
+    paymentPointer: {
+      relation: Model.HasOneRelation,
+      modelClass: PaymentPointer,
```

Comment:
can we change this to use the path based imports like in auth to prevent cyclic imports in the future.

Reply:
https://vincit.github.io/objection.js/guide/relations.html#require-loops
>In addition to objection's solutions, you can always organize your code so that such loops are not created.

If we really want it, we can open an issue to change all of the backend model relations in one go.